data/saas/config/appsflyer_config.yml

```diff
@@ -0,0 +1,94 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Appsflyer
```

Comment:
The preferred casing is "AppsFlyer", make sure to update it whenever it's used as a label or in a sentence.